README.md

```diff
@@ -0,0 +1,754 @@
+| `nonce` | `Bytes`                                   | Yes      | Nonce                                                       |
+| `meta`  | `{String : Any}`                          | No       | [Meta] (asserted, signed data) — is not delegated authority |
+| `nbf`   | `Integer` (53-bits[^js-num-size])         | No       | "Not before" UTC Unix Timestamp in seconds (valid from)     |
+| `exp`   | `Integer \| null` (53-bits[^js-num-size]) | Yes      | Expiration UTC Unix Timestamp in seconds (valid until)      |
+
+# Capability
+
+Capabilities are the semantically-relevant claims of a delegation. They MUST be presented as a map under the `cap` field as a map. This map is REQUIRED but MAY be empty. This MUST take the following form:
+
+| Field  | Type      | Required | Description                                                                                      |
+|--------|-----------|----------|--------------------------------------------------------------------------------------------------|
+| `sub`  | `DID`     | Yes      | The [Subject] that this Capability is about                                                      |
+| `cmd`  | `Command` | Yes      | The [Command] of this Capability                                                                 |
+| `pol`  | `Policy`  | Yes      | Additional constraints on eventual Invocation arguments, expressed in the [UCAN Policy Language] |
+
+Here is an illustrative example:
+
+``` js
+{
+  // ...
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp"
+  "cmd": "/blog/post/create",
+  "pol": [
+    ["==", ".status", "draft"],
+    ["every", ".reviewer", ["match", ".email", "*@example.com"]],
+    ["some", ".tags", 
+      ["or",
+        ["==", ".", "news"], 
+        ["==", ".", "press"]]]
+  ]
+}
+```
+
+## Subject
+
+The Subject MUST be the DID that initiated the delegation chain.
+
+For example:
+
+``` js
+{
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp",
+  // ...
+}
+```
+
+## Resource
+
+Unlike Subjects and Commands, Resources are semantic rather than syntactic. The Resource is the "what" that a capability describes.
+
+By default, the Resource of a capability is the Subject. This makes the delegation chain self-certifying.
+
+``` js
+{
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp", // Subject
+  // ...
+}
+```
+
+In the case where access to an [external resource] is delegated, the Subject MUST own the relationship to the Resource. The Resource SHOULD be referenced by a `uri` key in the relevant [Conditions], except where it would be clearer to do otherwise. This MUST be defined by the Subject and understood by the executor.
+
+``` js
+{
+  "sub": "did:key:z6MkiTBz1ymuepAQ4HEHYSF1H8quG5GLVVQR3djdX3mDooWp",
+  "cmd": "/crud/create",
+  "pol": [
+    ["==", ".url", "https://example.com/blog/"], // Resource managed by the Subject
+    // ...
+  ],
+  // ...
+}
+```
+
```

Comment:
This suggestion is pretty rough, but I think spec needs to cover commands and specifically aspects noted.

```suggestion

## Command

The [Command] MUST be a `/` delimited path describing set of commands delegated.

Delegation covers exact [Command] specified and all the commands described by a paths nested under that specified command. By definition `"/"` implies all of the command on a resource and SHOULD be used rarely and with a great care.

> ℹ️ Path based syntax is designed to support forward compatible protocol extensions - new capabilities could be introduced as command subpaths.
```

Reply:
> [!NOTE]
> I modified the syntax to use the built-in callouts, but essentially merged verbatim. Thanks!